cssom/style-sheet-interfaces-003.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSS Test: CSSOM StyleSheet Modify Rule List</title>
+    <link rel="author" title="Sendil Kumar N" href="mailto:sendilkumarn.opensource@gmail.com">
+    <link rel="help" href="http://www.w3.org/TR/cssom-1/#the-cssstylesheet-interface">
+    <link rel="help" href="http://www.w3.org/TR/cssom-1/#the-cssrule-interface">
+    <meta name="flags" content="dom">
+    <meta name="assert" content="StyleSheet and CSSStyleSheet objects have the properties specified in their interfaces">
+    <script src="/resources/testharness.js" type="text/javascript"></script>
+    <script src="/resources/testharnessreport.js" type="text/javascript"></script>
+    <style id="styleElement" type="text/css" media="all" title="internal style sheet" disabled="disabled">
+        * { margin: 0; padding: 0; }
```

Comment:
Maybe change the selector to something that matches nothing, and explain why this is necessary in the test; e.g.:
```
/* An initial style rule to test where the new rule is inserted relative to this one */
nosuchelement { color: red; }
```